services/frontend-v3/src/components/searchBar/SearchBarView.jsx

```diff
@@ -0,0 +1,107 @@
+import React from "react";
+import { injectIntl } from "react-intl";
+import { Form, Row, Select, Col, Button, Icon, Card } from "antd";
+
+import logo from "../sideNav/logo_v2.svg";
+
+const { Option } = Select;
+class SearchBarView extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      expand: false
+    };
+  }
+
+  render() {
+    const {
+      getFields,
+      getBasicField,
+      handleSearch,
+      handleReset,
+      toggle
+    } = this.props;
+    const { data } = this.props;
+
+    const searchLabel = this.props.intl.formatMessage({
+      id: "button.search",
+      defaultMessage: "Search"
+    });
+
+    return (
+      <Form onSubmit={handleSearch}>
+        <div
+          style={{
+            paddingTop: "80px",
+            paddingLeft: "20%",
+            paddingRight: "20%",
+            paddingBottom: "20px"
+          }}
+        >
+          <div
+            style={{
+              backgroundColor: "#001C1A",
+              borderRadius: "5px",
+              paddingTop: "80px",
+              paddingLeft: "80px",
+              paddingRight: "80px",
+              paddingBottom: "80px",
+              boxShadow: "5px 5px 5px #cccccc"
+            }}
+          >
+            <header
+              style={{
+                paddingBottom: "20px",
+                textAlign: "center"
+              }}
+            >
+              <img src={logo} alt="UpSkill Logo" style={{ height: "80px" }} />;
+            </header>
+            {/* Gets main basic search field and shows buttons beneath */}
+            <div style={{ paddingBottom: "20px" }}>{getBasicField(data)}</div>
```

Comment:
try to create a styles object and pass it into the style prop of each component. Yo can see the component TopNav or SideNav as an example. This is for consistency.